vue-components/src/components/ToggleButtonGroup.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+	<span class="wikit wikit-ToggleButtonGroup">
+		<slot />
+	</span>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+
+export interface ToggleButtonGroupInjection {
+	groupValue: ( () => string ) | null;
+	toggleListener: ( ( value: string ) => void ) | null;
+}
+
+export default Vue.extend( {
+	name: 'ToggleButtonGroup',
+	provide(): ToggleButtonGroupInjection {
+		return {
+			groupValue: (): string => this.value,
+			toggleListener: ( event: string ): void => {
+				this.$emit( 'input', event );
+			},
+		};
+	},
+	props: {
+		value: {
+			required: true,
+			type: String,
+		},
+	},
+} );
+</script>
+<style lang="scss">
+.wikit-ToggleButtonGroup :not(:last-child) {
+	border-inline-end: none;
+	border-start-end-radius: 0;
+	border-end-end-radius: 0;
+
+	// Chrome doesn't support logical border radius yet.
+	// TODO: Delete later.
+	border-top-right-radius: 0;
+	border-bottom-right-radius: 0;
+}
+
+.wikit-ToggleButtonGroup :not(:first-child) {
+	border-start-start-radius: 0;
+	border-end-start-radius: 0;
+}
+
+// Until https://github.com/csstools/postcss-logical/issues/27 is resolved
+.wikit-ToggleButtonGroup :not(:first-child):dir(ltr) {
+	border-top-left-radius: 0;
+	border-bottom-left-radius: 0;
+}
+
+.wikit-ToggleButtonGroup :not(:first-child):dir(rtl) {
+	border-top-right-radius: 0;
+	border-bottom-right-radius: 0;
+}
+
+.wikit-ToggleButtonGroup :not(:last-child):dir(ltr) {
+	border-top-right-radius: 0;
+	border-bottom-right-radius: 0;
+}
+
+.wikit-ToggleButtonGroup :not(:last-child):dir(rtl) {
+	border-top-left-radius: 0;
+	border-bottom-left-radius: 0;
+}
```

Comment:
@ladsgroup There is already code here to try and mitigate the issue you found (not sure if it was added by you if it was already here before because of the force pushing).

I think we should focus on ensuring the approach from line 50 onwards is working rather than repeat those rules as seen in lines 39 - 42. This is because the approach from line 50 onwards is closer to how the resulting code would look like after transpilation.

Also, instead of adding an inline to-do, it is preferable to add a task in the incoming column of the design system board, so that the fix will not be lost or fall out of sync with the code.

Reply:
I should remove the 39-42. I will make a ticket for it as well.

Reply:
Ticket is made: https://phabricator.wikimedia.org/T272866